src/build/options/build_options.rs

```diff
@@ -32,6 +32,11 @@ pub struct BuildOptions {
     ///
     /// It defaults to "en"
     pub default_language: String,
+
+    /// Whether to format the generated file or not (uses rustfmt).
+    ///
+    /// Defaults to false.
+    pub format: bool,
```

Comment:
Could you default this to true. It's always useful and should be opt-out in case it poses some problem.